frontend/packages/dev-console/integration-tests/features/Knative/usability-Enhancement-Eventing-ODC-3953.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Usability Enhancement for Eventing
+    User should be able to select BootStrapServers and Topics from provided options
+
+
+Background:
+   Given cluster with OpenShift Serverless Operator installed
+   And Kafka Operator installed
```

Comment:
we may need to create CR for `KEK` as well

Reply:
User has installed the Serverless Operator and kafka operator that means user has done all the prerequisites like creating serving, eventing and CR for Kafka.